testnet/stacks-node/src/burnchains/bitcoin_regtest_controller.rs

```diff
@@ -671,7 +793,15 @@ impl BitcoinRegtestController {
         tx.output
             .push(payload.recipient.to_bitcoin_tx_out(DUST_UTXO_LIMIT));
 
-        self.finalize_tx(&mut tx, DUST_UTXO_LIMIT, DUST_UTXO_LIMIT, utxos, signer, 1)?;
+        self.finalize_tx(
+            &mut tx,
+            DUST_UTXO_LIMIT,
+            0,
+            230,
```

Comment:
What's `230` here?

Reply:
`230` is the first estimated size we're using for the bitcoin `StackStx`  operations (it's a `#[cfg(test)]` method and used once, but I can use a constant if you prefer)

Reply:
Refactored in https://github.com/blockstack/stacks-blockchain/pull/2383/commits/4d576c9ffe82b31c8efe4b4659f30b6fa7e576cd